### PR TITLE
Update values.yaml

### PR DESCRIPTION
**Pull request by wooxus**
Fixing the TLS example to be the correct format: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls

### CODE REVIEW

**Comment by nineinchnick** [comment]
What's wrong with the current example? The order of properties in an object doesn't matter

**Comment by wooxus** [comment]
> What's wrong with the current example? The order of properties in an object doesn't matter

"secretName" does not get picked up when being passed as a list. I could not get a cert to work until I've removed the dash.

**Comment by nineinchnick** [comment]
Can you show the `ingress` section from your `values.yaml` file? The example might be confusing, but I believe it's correct, since the `tls` section is a list, and the example shows a single entry for it.

**Comment by wooxus** [comment]
> Can you show the `ingress` section from your `values.yaml` file? The example might be confusing, but I believe it's correct, since the `tls` section is a list, and the example shows a single entry for it.

If you try, it will fail. `hosts` should be a list and `secretName` a variable. So simple dash at the wrong place. Pls check the updated PR.

**Comment by nineinchnick** [comment]
Can you include the full `values.yaml` that fails? It would be easier to compare with the example.

File: charts/trino/values.yaml [hunk 1]
```diff
@@ -1001,8 +1001,8 @@ ingress:
   # @raw
   # Example:
   # ```yaml
-  #  - secretName: chart-example-tls
-  #    hosts:
+  #    secretName: chart-example-tls
```
[the remark this answers is not in the document]
Comment (wooxus) [reply charts/trino/values.yaml:1004]: indeed, it should be:

```
tls:
  secretName: chart-example-tls
- hosts:
    - chart-example.local
```

[the remark this answers is not in the document]
Comment (nineinchnick) [reply charts/trino/values.yaml:1004]: No, `secretName` should be in the same `tls` array item as `hosts`, so:
```
tls:
- secretName: chart-example-tls
  hosts:
    - chart-example.local
```
The example already looks like this - it only does not include the `tls` key.

**Pull request closed without merging**